index.html

```diff
@@ -110,22 +118,66 @@ <h4 class="backyear">Backend Dev</h4>
           <h4 class="backyear">2015</h4>
         </div>
         <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required.
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
         </p>
         <ul class="tags">
           <li class="tag">html</li>
           <li class="tag">css</li>
           <li class="tag">javascript</li>
         </ul>
         <div class="action">
-          <button type="button">See Project</button>
+          <button data-id="1" type="button" id="tonic-modal-open">See Project</button>
+        </div>
+      </div>
+
+      <!-- Tonic card modal -->
+      <div class="tonic-card-modal-container" id="tonic-card-modal-container">
+        <div class="tonic-card-modal">
+          <h1 class="tonic-title">Tonic Modal</h1>
+          <div class="tonic-stacks">
+            <ul>
+              <li class="tonic-stack1"></li>
+              <li class="tonic-stack2"></li>
+              <li class="tonic-stack3"></li>
+            </ul>
+          </div>
+          <div class="tonic-image-holder">
+            <img src="./image/desktopad1.png" alt="tonic" class="tonic-image" />
+          </div>
+          <div class="tonic-contents">
+            <p class="tonic-description"></p>
+            <div class="tonic-subcontent">
+              <div class="technologies">
+                <ul>
+                  <li class="tonic-html"></li>
+                  <li class="tonic-css"></li>
+                  <li class="tonic-javascript"></li>
+                  <li class="tonic-ruby">ruby on rails</li>
+                </ul>
+              </div>
+              <div class="network">
+                <a href="https://zemola.github.io/MY-PORTFOLIO/" target="_blank" rel="noreferrer noopener"
+                ><span>See Live</span>
+                  <svg class="w-6 h-6" fill="none"
+                    stroke="currentColor" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+                    <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+                      d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14"></path>
+                  </svg></a>
+                <a href="https://github.com/zemola/MY-PORTFOLIO.git" target="_blank" rel="noreferrer noopener">
+                  <span>See Source</span>
+                  <i class="fab fa-github-square network-link"></i>
+                </a>
```

Comment:
Please make sure this also match your preferred template design✔👌
- Your Design
![screenshot (4)](https://user-images.githubusercontent.com/44624138/141211854-54c050cd-2532-4732-a7d2-de57843953d8.png)
- Figma Design
![screenshot (5)](https://user-images.githubusercontent.com/44624138/141211864-18426599-ef92-45e8-b7cd-f5492bf1ae71.png)